modules/tinymce/src/plugins/media/main/ts/core/FilterContent.ts

```diff
@@ -5,64 +5,54 @@
  * For commercial licenses see https://www.tiny.cloud/
  */
 
-import { Arr } from '@ephox/katamari';
+import { Arr, Obj } from '@ephox/katamari';
 
 import Editor from 'tinymce/core/api/Editor';
 import AstNode from 'tinymce/core/api/html/Node';
-import Tools from 'tinymce/core/api/util/Tools';
 
 import * as Nodes from './Nodes';
 import * as Sanitize from './Sanitize';
 
 declare let unescape: any;
 
 const setup = (editor: Editor): void => {
-  editor.on('preInit', () => {
-    // Make sure that any messy HTML is retained inside these
-    const specialElements = editor.schema.getSpecialElements();
-    Tools.each('video audio iframe object'.split(' '), (name) => {
-      specialElements[name] = new RegExp('<\/' + name + '[^>]*>', 'gi');
+  editor.on('PreInit', () => {
+    const { schema, serializer, parser } = editor;
+    // Set browser specific allowFullscreen attribs as boolean
+    const boolAttrs = schema.getBoolAttrs();
+    Arr.each('webkitallowfullscreen mozallowfullscreen'.split(' '), (name) => {
```

Comment:
I wonder how long we need to support these. But I guess Flash is dead as well this plugin really needs to be rewritten.

Reply:
Yeah I seriously considered removing these tbh and you're right this entire plugin is rather messy or likely has obsolete functionality given todays standards.